.github/workflows/release.yml

```
@@ -0,0 +1,132 @@
+name: Release
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)